tests/Api/Response/ResponseParserTest.php

```diff
@@ -62,6 +63,122 @@ public function testParse()
         ]);
     }
 
+    /**
+     * @dataProvider smartDidYouMeanProvider
+     *
+     * @param string $response
+     * @param array|null $expectedResult
+     */
+    public function testSmartDidYouMeanParsing(string $response, $expectedResult)
+    {
+        $responseMock = $this->getMockBuilder(Response::class)->disableOriginalConstructor()->setMethods(null)->getMock();
+        /** @var ResponseParser|MockObject $responseParserMock */
+        $responseParserMock = $this->getResponseParserMock(['createResponseObject', 'handleLandingPage']);
+        $responseParserMock->expects($this->any())->method('createResponseObject')->willReturn($responseMock);
+
+        $results = $responseParserMock->parse($response);
+
+        $this->assertEquals($results->getData(Response::DATA_SMART_DID_YOU_MEAN), $expectedResult);
+    }
+
+    public function smartDidYouMeanProvider()
+    {
+        return [
+            'No Smart Did-You-Mean data provided' => [
+                '<?xml version="1.0"?>
+                <searchResult>
+                    <servers>
+                        <frontend>frontend.findologic.com</frontend>
+                        <backend>backend.findologic.com</backend>
+                    </servers>
+                    <query>
+                        <limit first="0" count="10"/>
+                        <queryString>Test</queryString>
+                        <searchedWordCount>1</searchedWordCount>
+                        <foundWordCount>1</foundWordCount>
+                    </query>
+                    <results><count>0</count></results>
+                    <products/>
+                    <filters/>
+                </searchResult>'
+                ,
```

Comment:
This doesn't have to have its own line :wink: 
Put it on the same line as `</searchResult>'` (like it's done for the other scenarios).